themes/math_research_center/scss/molecules/hover-menu.scss

```diff
@@ -129,7 +130,79 @@
   }
 }
 
-// Styles for no javascript
+// Subnavigation - no JavaScript
+// No JavaScript is applied to the menu when
+// it does not appear in the Header region.
+.js .decanter-width-one-fourth {
```

Comment:
I dont think we would want to target this incase we change the the width of that sidebar.

Reply:
For sure! Was trying to find the best unique identifier. Now that we have removed the JS from every region other than the header, perhaps I replace that with .main-container.